lib/clientsidescripts.js

```diff
@@ -36,6 +43,83 @@ function wrapWithHelpers(fun) {
       '  return (' + fun.toString() + ').apply(this, arguments);');
 }
 
+/* Tests if an ngRepeat matches a repeater
+ *
+ * @param {string} ngRepeat The ngRepeat to test
+ * @param {string} repeater The repeater to test against
+ * @param {boolean} exact If the ngRepeat needs to be more or less exactly the
```

Comment:
Well, it removes ` track by ` and ` as ` and everything after `|` or `=`.  Is it still *exactly* the repeater?  I don't actually understand ng1 well enough to know which parts of the `ng-repeat` count.